app/Resources/views/macro/messages.html.twig

```diff
@@ -0,0 +1,25 @@
+{#
+   Macros are comparable with functions in regular programming languages.
+   They are useful to put often used HTML idioms into reusable elements
+   to not repeat yourself.
+   See http://twig.sensiolabs.org/doc/tags/macro.html
+#}
+{% macro render_flash_messages() %}
+    {% from _self import alert %}
+
+    <div class="messages">
+        {% for flash_message in app.session.flashBag.get('success') %}
```

Comment:
well, you are asking to read from the session. currently, Symfony requires to start the session to read from it. There is proposal to change it, but it cannot be considered as a bugfix (it is a big refactoring), and I'm not sure it is ready.